packages/core/src/routes/applications/utils.ts

```diff
@@ -7,4 +7,8 @@ export const buildProtectedAppMetadata = ({ host, origin }: { host: string; orig
     sessionDuration: defaultProtectedAppSessionDuration,
     pageRules: defaultProtectedAppPageRules,
   },
+  oidcClientMetadata: {
```

Comment:
double check the function `buildProtectedAppMetadata` is used dynamically for protected apps (instead of inserting db). better to have some comments for this func

Reply:
Yes, comments added